cachito/workers/pkg_managers/pip.py

```diff
@@ -1987,7 +1987,7 @@ def _push_downloaded_requirement(requirement, pip_repo_name, raw_repo_name):
         dest_dir, filename = requirement["raw_component_name"].rsplit("/", 1)
         try:
             upload_raw_package(raw_repo_name, requirement["path"], dest_dir, filename, True)
-        except (NetworkError, ValueError):
+        except (UploadError, NetworkError, ValueError):
```

Comment:
Here it looks like we can remove both `ValueError` and `NetworkError`

Reply:
Only here, not for the other one? As far as I can tell, `ValueError` is only raised for invalid usage of the method